### PR TITLE
[bug-398]: Support storage system names/IDs with capital letters

### PR DESCRIPTION
**Pull request by atye**
# Description
`viper` returns configuration values in lowercase even if the actual value has uppercase characters so we need to read the storage-systems.yaml file directly and extract the contents. Also removed the `return` statements for each storage type update so if PowerFlex systems update fails (for example), PowerMax and PowerScale will still be attempted.

Changed the default `LOG_LEVEL` of the RPM installation to `debug`.

Added missing error handling when asking OPA for a decision.

Changed some error responses to lowercase to be consistent.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/398 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

Unit tests for reading the configuration file. Manually verified that connecting to a PowerScale cluster with capital letters in the cluster name works with both deployment paths.

### CODE REVIEW

**Review by shaynafinocchiaro — APPROVED** [review]

File: deploy/install.go [hunk 1]
```diff
@@ -640,7 +640,7 @@ func (dp *DeployProcess) WriteConfigMapManifest() {
 
 	settings := dp.cfg.AllSettings()
 
-	logLevel := "info"
+	logLevel := "debug"
```
Comment (sharmilarama) [inline deploy/install.go:643]: Why do we want this? The default is usually info, debug can be set through the ConfigMap if additional logs are required.

Comment (atye) [reply deploy/install.go:643]: I noticed that the drivers and the Helm deployment of Authorization are debug by default so I thought we can be consistent.

Comment (sharmilarama) [reply deploy/install.go:643]: I see. I don't think we have a set standard, we can leave it in to be consistent.

**Review by sharmilarama — DISMISSED** [review]

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by sharmilarama — APPROVED** [review]

**Comment by atye** [comment]
run e2e test

**Pull request merged**